pdfminer/pdfpage.py

```diff
@@ -120,15 +121,18 @@ def search(obj, parent):
     @classmethod
     def get_pages(cls, fp,
                   pagenos=None, maxpages=0, password='',
-                  caching=True, check_extractable=True):
```

Comment:
Set default to `False` instead of removing.

Reply:
Done